src/settings_gui.cpp

```diff
@@ -140,6 +142,52 @@ void ShowBaseSetTextfileWindow(TextfileType file_type, const TBaseSet* baseset,
 	new BaseSetTextfileWindow<TBaseSet>(file_type, baseset, content_type);
 }
 
+std::vector<int> refresh_rates = { 30, 60, 75, 90, 100, 120, 144, 240 };
+
+// We want 60Hz as the default for this pointer, though it gets overriden with whatever is in the config.
+int *current_refresh_rate = &refresh_rates[1];
+
+std::string hertz = "Hz";
+std::string current_refresh_rate_str = std::to_string(*current_refresh_rate) + hertz;
+
+// Add the refresh rate from the config and the refresh rates from all the monitors to
+// our list of refresh rates shown in the GUI.
+void addRefreshRatesAndSelect() {
+	// std::unordered_set so we can efficiently add all monitor refresh rates to this
+	// set without duplicates, as it has near constant lookup times.
```

Comment:
This comment seems to explain how std:: works. Not sure that is really useful in this context, especially as the list is really small. I love comments, but reading comments that didn't add context .. not so much :D

Reply:
It was more meant as clarification of why I used that instead of just using another `std::vector<int>` or simply appending to the already existing vector.

Reply:
Basically: `/* Using a set to make sure no duplicated refresh rates are in the  list. */`
? :D

That the lookup time is near constant doesn't really add value (to me).

Reply:
Also, capitalisation of functions.